src/dataconnect/schemaMigration.ts

```diff
@@ -75,7 +113,8 @@ export async function migrateSchema(args: {
     /* linkIfNotConnected=*/ true,
   );
 
-  setCompatibleMode(schema, databaseId, instanceName);
+  const valMode = experiments.isEnabled("fdccompatiblemode") ? "COMPATIBLE" : "STRICT";
```

Comment:
Nit: prefer validationMode over valMode for clarity when we reference it again 20 lines later